engine/pom.xml

```diff
@@ -229,9 +224,7 @@
                             org.joda.time.*;version="[2.0.0,3.0.0)";resolution:=optional,
                             org.jsoup.*;version="[1.5.2,2.0.0)";resolution:=optional,
                             javax.money;version="[1.0.0,2.0.0)";resolution:=optional,
-                            com.thoughtworks.paranamer.*;version="[2.5.5,3.0.0)";resolution:=optional,
-                            org.codehaus.groovy.jsr223;version="[2.4.0,3.0.0)";resolution:=optional,
```

Comment:
hmm so now that those ScriptEvaluatorFactories are not inside of HV engine there's no need to import scripting packages, that's really nice!